### ISSUE TITLE
bluetooth earphone's voice disappear when switch to open safari to surf on the internet repeatedly

### ISSUE THREAD

**Issue by liangtb** [lhc70000/iina#1076]
**System and IINA version:**

- macOS 10.13
- IINA 0.0.12

**Expected behavior:**

**Actual behavior:**

- Crash report:
```

```

- MPV log:
```

```

**Steps to reproduce:**

- [x] MPV does not have this problem.

**How often does this happen?**
bluetooth earphone's voice disappear when switch to open safari to surf on the internet repeatedly
especially with bose qc30...which well-functional in sierra and other device such as mate9 or iphone6plus

**Comment by saagarjha** [comment]
Are you, by chance, playing videos in Safari?

**Comment by liangtb** [comment]
yes,when play videso in safari, it must happen, when surf on the internet, maybe okay.
